src/test/java/com/scaleunlimited/flinkcrawler/functions/TimerTest.java

```diff
@@ -0,0 +1,105 @@
+package com.scaleunlimited.flinkcrawler.functions;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.flink.api.common.typeinfo.BasicTypeInfo;
+import org.apache.flink.runtime.checkpoint.OperatorSubtaskState;
+import org.apache.flink.streaming.api.operators.KeyedProcessOperator;
+import org.apache.flink.streaming.runtime.streamrecord.StreamRecord;
+import org.apache.flink.streaming.util.KeyedOneInputStreamOperatorTestHarness;
+import org.apache.flink.streaming.util.OneInputStreamOperatorTestHarness;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.scaleunlimited.flinkcrawler.tools.TimerTool;
+
+public class TimerTest {
```

Comment:
This is just to validate that timers get saved with state, yes? Do we still want/need it?